apps/judicial-system/web/src/routes/Prosecutor/Overview/Overview.tsx

```diff
@@ -326,8 +327,26 @@ export const Overview: React.FC = () => {
                     </Box>
                   )}
                 </AccordionItem>
+                {workingCase.files && (
+                  <AccordionItem
+                    id="id_5"
+                    label={`Rannsóknargögn ${`(${workingCase.files.length})`}`}
+                    labelVariant="h3"
+                  >
+                    {workingCase.files?.map((file, index) => (
+                      <Box marginBottom={3}>
+                        <CaseFile
+                          name={`${index + 1}. ${file.name}`}
+                          size={file.size}
+                          uploadedAt={file.modified}
```

Comment:
I suggest you use "created" instead of "modified". Currently, they are the same, but later we will be updating the entries in the "file" table and then "modified" will not reflect the upload time.

Reply:
Cool, thanks, fixed 🙏